.github/workflows/release.yml

```diff
@@ -0,0 +1,136 @@
+## This workflow is triggered by a release being created.
+name: Terminus Release
+run-name: On Release, update homebrew and .deb package to ${{ inputs.deploy_target }} by @${{ github.actor }}
+on:
+  ## We have to explicitly tag a release and create a release
```

Comment:
There used to be an issue where self:update malfunctioned because a release was created without a .phar. See BUGS-4390

I think this may reintroduce that bug

Reply:
So I moved the "merge" build from the MERGE action to the TAG action. Anytime any commit is tagged, a build will be created.